libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -219,6 +358,18 @@ AP_GPS_UBLOX::_request_next_config(void)
             }
         }
         break;
+    case STEP_RTK_MOVBASE:
+        if (_hardware_generation >= UBLOX_F9) {
```

Comment:
Are you aware that `UBLOX_UNKNOWN_HARDWARE_GENERATION` is the highest number in the enum? We should be checking that we know the generation as well.

Reply:
fixed, added new supports_F9_config() private function